.github/workflows/ci.yaml

```
@@ -311,21 +331,27 @@ jobs:
         echo "##[group]kubectl get clusterworkloadresourcemappings.servicebinding.io"
           kubectl get clusterworkloadresourcemappings.servicebinding.io
         echo "##[endgroup]"
-        echo "##[group]kubectl get servicebindings.servicebinding.io"
-          kubectl get servicebindings.servicebinding.io
+        echo "##[group]kubectl get servicebindings.servicebinding.io -n servicebindings-cts"
+          kubectl get servicebindings.servicebinding.io -n servicebindings-cts
         echo "##[endgroup]"
-        echo "##[group]kubectl describe servicebindings.servicebinding.io"
-          kubectl describe servicebindings.servicebinding.io
+        echo "##[group]kubectl describe servicebindings.servicebinding.io -n servicebinding-cts"
+          kubectl describe servicebindings.servicebinding.io -n servicebindings-cts
         echo "##[endgroup]"
         echo "##[group]kapp list -A"
           ${KAPP} list -A
         echo "##[endgroup]"
         echo "##[group]kubectl get all -n servicebinding-system"
           kubectl get all -n servicebinding-system
         echo "##[endgroup]"
+        echo "##[group]kubectl get all -n servicebinding-system"
+          kubectl get all -n servicebindings-cts
```

Not sure which direction is the typo, but I assume this was intended

```suggestion
        echo "##[group]kubectl get all -n servicebindings-cts"
          kubectl get all -n servicebindings-cts
```

---

Fixed with 025848b.